packages/richtext-lexical/src/exports/react/components/RichText/converter/converters/upload.tsx

```diff
@@ -8,34 +8,29 @@ export const UploadJSXConverter: JSXConverters<SerializedUploadNode> = {
   upload: ({ node }) => {
     // TO-DO (v4): SerializedUploadNode should use UploadData_P4
     const uploadDocument = node as UploadDataImproved
-    if (typeof uploadDocument.value !== 'object') {
+    if (typeof uploadDocument?.value !== 'object') {
```

Comment:
I guess ts was asking for optional chaining here? any reason why it is needed now? afaik value should be defined